react/src/components/toggle/SprkToggle.js

```diff
@@ -35,15 +36,15 @@ class SprkToggle extends Component {
       titleAddClasses,
       iconAddClasses,
       toggleIconName,
-      contentId,
       ...other
     } = this.props;
     const { isOpen, height } = this.state;
 
-    const containerClasses = classnames(
-      'sprk-c-Toggle',
-      additionalClasses
-    );
+    const contentId = other.contentId
```

Comment:
Let's make the same changes on this file based on our conversation on SparkMastheadAccordionItem